File: Curso_POO/Python/uberVan.py
from car import Car

class UberVan(Car):
    typeCarAccepted=[]
    seatmaterial=[]
    __passenger=int

    def __init__(self, license, driver,typeCarAceepted,seatmaterial):
        super().__init__(license, driver)
        self.typeCarAceepted=typeCarAceepted
        self.seatmaterial=seatmaterial

File: Curso_POO/Java/UberVan.java
import java.util.ArrayList;
import java.util.Map;

class UberVan extends Car {
    Map<String, Map<String,Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial; Integer passenger;

    public UberVan(String license, Account driver){
        super(license, driver);
    
    }

    
    
    void printDataCar() {
        if(passenger !=null){
            System.out.println("License: "+license+" Driver: "+driver.name + " Passenger: "+passenger); 
        }
        
    }
    public void setPassenger(Integer passenger){
        if(passenger==6){
         this.passenger=passenger;
        }else{
         System.out.println("Necesitas asignar 6 pasajeros");
        }  
    }
    
   
   
   
   
   
    public Integer getPassenger(){
        return passenger;
    }
    
}